schemas/context/experienceevent.example.1.json

```diff
@@ -114,5 +114,48 @@
   },
   "xdm:marketing": {
     "xdm:trackingCode": "marketingcampaign111"
-  }
+  },
+  "xdm:profileStitching": [
+    {
+      "xdm:profileStitchID": {
+        "@id": "https://data.adobe.io/entities/profileStitchIdentity/1",
+        "xdm:namespace": {
+          "xdm:code": "AAM"
+        }
+      },
+      "xdm:version": "1.0",
+      "xdm:endUserIds": {
+        "https://ns.adobe.com/experience/mcid": {
+          "xdm:id": "https://data.adobe.io/entities/identity/92312748749128",
+          "additionalIDs": [
+            "https://data.adobe.io/entities/identity/62312748749321",
+            "https://data.adobe.io/entities/identity/49312748749132"
+          ],
+          "xdm:namespace": {
+            "xdm:code": "ECID"
+          }
+        }
+      }
+    }
+  ],
+  "xdm:segmentMemberships": [
+    {
+      "xdm:segmentID": {
+        "@id": "https://data.adobe.io/entities/identity/92312748749128",
+        "xdm:namespace": {
+          "xdm:code": "AAM"
+        }
+      },
+      "xdm:profileStitchID": {
```

Comment:
Is there a reason to add this to each segment realization and each ExperienceEvent? This information can be looked up on demand from the segment metadata.

Reply:
@mohrahit, another alternative existed also. That was to have segmentmembership be a property under each profile stitch here in ExperienceEvent. That felt too nested and more importantly, assumed that all segmentation happened with profile stitching.

Reply:
@mohrahit, are you okay with this approach?